cruise/CruiseInformation.csv

```diff
@@ -194,4 +222,4 @@ WJ20200323,M/V Warren Jr.,2020-03-23T00:00:00,2020-03-27T23:59:00,Pioneer 13c AU
 AR-45,R/V Neil Armstrong,2020-04-17T00:00:00,2020-05-07T23:59:00,Pioneer 14
 SKQ202009S,R/V Sikuliaq,2020-05-21T00:00:00,2020-06-06T23:59:00,Papa 8
 AR46-01,R/V Neil Armstrong,2020-05-22T00:00:00,2020-06-21T23:59:00,Irminger 7
-AR-48,R/V Neil Armstrong,2020-09-30T00:00:00,2020-10-20T23:59:00,Pioneer 15
+AR-48,R/V Neil Armstrong,2020-09-30T00:00:00,2020-10-20T23:59:00,Pioneer 15
```

Comment:
Not sure that we should add this one now due to uncertainty in which cruises will occur & changes in cruise IDs.

Reply:
I didn't actually make a change here. It's just showing a change because this is the new end of the file. This cruise is already in the list.

Reply:
Ah - so I probably added that when we did all the 2020 cruises before all this 'stuff' happened.  Ok - there will definitely be updates in the future for everything in 2020. Good for now though since that's all still in flux.